### PR TITLE
Implement zmb get_test_params

### PR DESCRIPTION
**Pull request by mattias-p**
## Purpose

Add another RPC API method to zmb.

## Context

N/A

## Changes

Add `zmb get_test_params`.

Rename the `--testid` parameter to `--test-id` for the `test_progress` and `get_test_results` commands. The old parameter name still works but emits a deprecation warning.

## How to test this PR

Start a test and verify that the output from `get_test_params` looks reasonable:
```sh
testid="$(zmb start_domain_test --domain example | jq -r .result)"
zmb get_test_params --test-id "$testid" | jq .
```

Verify that zmtest still works:
```sh
zmtest example.com
```

### CODE REVIEW

**Review by matsduf — DISMISSED** [review]
It looks fine. It works as expected. I think we should include all API methods into `zmb` (but not in this PR).

**Comment by matsduf** [comment]
@pnax, can you review this?

File: script/zmb [hunk 1]
```diff
@@ -270,6 +270,34 @@ sub cmd_test_progress {
 }
 
 
+=head2 get_test_params
+
+ zmb [GLOBAL OPTIONS] get_test_params [OPTIONS]
+
+ Options:
+   --test-id TEST_ID
```
Comment (pnax) [inline script/zmb:278]: It looks like other methods use `--testid` (see `test_progress` and `get_test_results`). Could you change this one to be consistent with the other ? (Or the other way around).

**Review by pnax — CHANGES_REQUESTED** [review]

File: script/zmb (hunk 1 above)
Comment (matsduf) [reply script/zmb:278]: Outside this change I find two `--testid` and one `--test-id`. I agree that making them consistent would be good. I might also affect `zmtest` for which `zmb` is a backend.

Comment (mattias-p) [reply script/zmb:278]: I agree that this sucks, but I actually made it so in order to be consistent with the [RPCAPI](https://github.com/zonemaster/zonemaster-backend/blob/master/docs/API.md#api-method-get_test_params).

Comment (matsduf) [reply script/zmb:278]: I think we can start by making it consistent in zmb because it is more likely that you do things manually there. Or support both.

Comment (pnax) [reply script/zmb:278]: Could you then update all the other one to `--test-id` in this PR, so they all evolve together ?

Comment (mattias-p) [reply script/zmb:278]: I just checked and it's inconsistent all over the place. Maybe we should start migrating to `testid` for test ids across the board. I'll deprecate all `test-id`, `test_id` and `id` that I can find.

Comment (mattias-p) [reply script/zmb:278]: I created #790 to better facilitate such a migration. The migration itself is out of scope for this PR.

**Comment by mattias-p** [comment]
I renamed the --testid parameters so they're all named --test-id now.

@pnax @matsduf Please re-review.

**Comment by mattias-p** [comment]
I'll make another update that does away with the deprecations in zmb. It clearly states that it's interface is unstable so we may well make use of that.

**Review by matsduf — APPROVED** [review]

**Comment by matsduf** [comment]
@pnax, are your comments on this PR adressed by the last updates?

**Review by pnax — APPROVED** [review]
yes, this looks good to me.

**Comment by pnax** [comment]
## v2021.2 testing

LGTM

**Pull request merged**